src/components/DpSearchField/DpSearchField.vue

```diff
@@ -1,15 +1,27 @@
 <template>
-  <span :class="{ 'inline-block w-full': inputWidth !== ''}">
+  <span
+    class="inline-flex"
+    :class="{ 'w-full': inputWidth !== ''}">
     <dp-resettable-input
       id="searchField"
       data-cy="searchField"
       :class="cssClasses"
-      :input-attributes="{ placeholder: translations.search, type: 'search' }"
+      :input-attributes="{ placeholder: Translator.trans('search'), type: 'search'}"
       @reset="handleReset"
       @enter="handleSearch"
-      v-model="searchTerm" /><!--
+      v-model="searchTerm">
+      <!-- Slot for additional buttons -->
+      <slot />
+    </dp-resettable-input>
 
- --><dp-button
+    <dp-button
+      class="search rounded-r-md rounded-l-none"
+      data-cy="handleSearch"
+      hide-text
+      icon="search"
+      :text="Translator.trans('searching')"
```

Comment:
please don't use the Translator

Reply:
Done in 01e6c4d4d2a590291b0dffb2a5165249ae199b67